### PR TITLE
fix(routes): get blob upload session should include 'bytes=' in Range…

### PR DESCRIPTION
**Pull request by peusebiu**
… header

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/api/routes.go [hunk 1]
```diff
@@ -1189,7 +1189,7 @@ func (rh *RouteHandler) GetBlobUpload(response http.ResponseWriter, request *htt
 	}
 
 	response.Header().Set("Location", getBlobUploadSessionLocation(request.URL, sessionID))
-	response.Header().Set("Range", fmt.Sprintf("0-%d", size-1))
+	response.Header().Set("Range", fmt.Sprintf("bytes=0-%d", size-1))
```
Comment (andaaron) [inline pkg/api/routes.go:1192]: Not sure about this. We made some changes in https://github.com/project-zot/zot/pull/596/files
See that discussions.

**Comment by rchincha** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range
<unit> doesn't look optional. So would just make this PR official.
Can we also look at all Range responses and fix them pls.

**Review by jdolitsky — APPROVED** [review]

**Comment by andaaron** [comment]
The range responses were modified as part of https://github.com/project-zot/zot/pull/596/files
And there was a previous discussion on the dist spec in https://github.com/opencontainers/distribution-spec/pull/203

**Comment by jdolitsky** [comment]
i think this can be closed, we just merged https://github.com/opencontainers/distribution-spec/pull/401 with missing bytes prefix

**Pull request closed without merging**